### ISSUE TITLE
Unexpected Behavior when saving default language and translations of a field

### ISSUE THREAD

**Issue by nothdurfterf** [cakephp/cakephp#9695]
This is a (multiple allowed):
* [x] bug
* [ ] enhancement
* [ ] feature-discussion (RFC)

* CakePHP Version: 3.3.7
* Platform and Target: Mac OSX Sierra, MAMP PRO with PHP 7.0.12

### What you did
When sending a Form with either add or edit in the default language:

```
$this->Form->input('name', ['label' => __('Name')]);
$this->Form->input('_translations.en.name', ['label' => __('Name EN')]);
```

Controller
```
$meansOfTransport = $this->MeansOfTransport->patchEntity($meansOfTransport, $this->request->data, [
	'translations' => TRUE
]);
$this->MeansOfTransport->save($meansOfTransport)
```

Configuraton of MeansOfTransport Table:
```
$this->addBehavior('Translate', [
	'fields' => [
			'name' 
	], 
	'translationTable' => 'LanguageValues', 
]);
```

### What happened
When saving the data using the built in TranslateBehavior and TranslateTrait in models the value for the default language is saved as a record in the translationstable with "locale" set to the default language. The value of the field "name" in parent record is not saved or modified. For "add" the value ist empty, for "edit" the value stays the same as it was before the change.

### Expected Behavior
The value for the default language should be saved into the parent record not into the translationstable.
A workaround would be, saving the record without the translations first, and then saving the translations without the presence of the value for the default language.
```
$meansOfTransport = $this->MeansOfTransport->patchEntity($meansOfTransport, array_diff_key($this->request->data, [
	'_translations' => TRUE 
]));
if($this->MeansOfTransport->save($meansOfTransport)){
	$this->MeansOfTransport->save($this->MeansOfTransport->patchEntity($meansOfTransport, array_intersect_key($this->request->data, [
		'_translations' => TRUE 
	]), [
	'translations' => TRUE 
]));
```

### The problem seems to be
in **TranslateBehavior.php** Line 307 - 340
On line 328 the record with the default language in the translation table is created. Is this the expected behavior?
```
        $model = $this->_config['referenceName'];
        $preexistent = $this->_translationTable->find()
            ->select(['id', 'field'])
            ->where([
                'field IN' => $fields,
                'locale' => $locale,
                'foreign_key' => $key,
                'model' => $model
            ])
            ->bufferResults(false)
            ->indexBy('field');

        $modified = [];
        foreach ($preexistent as $field => $translation) {
            $translation->set('content', $values[$field]);
            $modified[$field] = $translation;
        }

        $new = array_diff_key($values, $modified);
        foreach ($new as $field => $content) {
            $new[$field] = new Entity(compact('locale', 'field', 'content', 'model'), [
                'useSetters' => false,
                'markNew' => true
            ]);
        }

        $entity->set('_i18n', array_merge($bundled, array_values($modified + $new)));
        $entity->set('_locale', $locale, ['setter' => false]);
        $entity->dirty('_locale', false);

        foreach ($fields as $field) {
            $entity->dirty($field, false);
        }
```

The last 3 lines cause the default language not to be saved in the parent record

**Comment by Xety** [comment]
Hey,

This bug has already been reported #9688  and a fix has already been merged #9692 . The fix will be available in the 3.3.8.

**Comment by ADmad** [comment]
Duplicate of #9688.  In future please search existing issues before opening new ones :)
